src/client/pages/AssessmentSection/Descriptions/Description/DataSources/DataSourceColumn/ColumnTypeOfDataSource.tsx

```diff
@@ -12,18 +12,34 @@ type Props = {
   onChange: (key: string, value: string) => void
 }
 
+type Item = {
+  label: string
+  value: string
+}
+
 const ColumnTypeOfDataSource: React.FC<Props> = (props: Props) => {
   const { dataSource, disabled, onChange } = props
   const { t } = useTranslation()
-  const _onChange = (value: string) => onChange('type', value)
+  const _onChange = ({ value }: Item) => onChange('type', value)
+
+  const items = useMemo(() => {
+    return Object.keys(dataSourceType).map((type) => {
+      return {
+        label: t(`dataSource.${type}`),
+        value: type,
+      }
+    })
+  }, [t])
+
   return (
     <DataColumn className="data-source-column">
       <Autocomplete
+        labelKey="label"
         withArrow
         disabled={disabled}
         onSave={_onChange}
-        value={dataSource.type}
-        items={Object.keys(dataSourceType).map((type) => t(`dataSource.${type}`))}
+        value={dataSource.type ? t(`dataSource.${dataSource.type}`) : ''}
```

Comment:
why is the value the label ? shouldn't the value be the key?

Reply:
it is the _displayValue_ ; the value shown

used here eg. to highlight match
https://github.com/openforis/fra-platform/blob/2c7ca8604c63b586e77cf783702400d5e71f1353/src/client/components/Autocomplete/Autocomplete.tsx#L25-L27